spec/spec_helper.rb

```diff
@@ -6,6 +6,8 @@
 
 Dir['./spec/support/**/*.rb'].each { |file| require file }
 
+ENV["TESTING"] = "1"
```

Comment:
It looks like we use this env flag twice in suspenders' code, to do the same `PATH` trick. Do you think we should instead forget about that variable, and modify the path directly in here? We have only two bin scripts there, and we don't want to hit the real commands while testing, so I'd probably do that.